tools/WinObjC.Tools/WinObjC.Tools.nuproj

```diff
@@ -43,7 +42,7 @@
     <None Include="Readme.txt">
       <IncludeInPackage>true</IncludeInPackage>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
+    </PackageFile>
```

Comment:
what is going on here? This is invalid XML. 

How are you verifying your changes before you push?

Reply:
I was checking each nuget package's content before pushing. It was late and I must have not checked this one.